pvc-autoresizer.yaml

```
@@ -0,0 +1,63 @@
+package:
+  name: pvc-autoresizer
+  version: 0.17.2
+  epoch: 0
+  description: Auto-resize PersistentVolumeClaim objects based on Prometheus metrics
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: eaa64de023df465ae879623b82427bd324dd25d8
+      repository: https://github.com/topolvm/pvc-autoresizer
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/net@v0.36.0
+        golang.org/x/oauth2@v0.27.0
+
+  - uses: go/build
+    with:
+      packages: ./cmd
+      output: pvc-autoresizer
+
+subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}
+          ln -sf /usr/bin/pvc-autoresizer ${{targets.subpkgdir}}/pvc-autoresizer
+
+test:
+  pipeline:
+    - name: Help tests for pvc-autoresizer
+      runs: |
+        pvc-autoresizer --help
+    - runs: |
+        # We expect the below command to fail.
+
+        export PROMETHEUS_URL="http://invalid-prometheus-server/api/v1/query"
+        export KUBERNETES_SERVICE_HOST=127.0.0.1
+        export KUBERNETES_SERVICE_PORT=8080
+
+        # Run pvc-autoresizer with the invalid URL
+        pvc-autoresizer --prometheus-url=$PROMETHEUS_URL --namespaces=default > /tmp/negative_output.log 2>&1 || true
+
+        cat /tmp/negative_output.log
+
+        if grep -q "unable to load in-cluster config" /tmp/negative_output.log; then
+          echo "Negative test passed: Expected error message found."
+        else
+          echo "Negative test failed: Expected error message not found."
+          exit 1
+        fi
```

Let's make this simpler:

```suggestion
	set -e # We expect the below command to fail.
        pvc-autoresizer --prometheus-url=[$PROMETHEUS_URL](http://invalid-prometheus-server/api/v1/query) --namespaces=default 2>&1 | grep -i "unable to load in-cluster config"
```